mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -96,6 +98,7 @@ def create(self, validated_data):
 
 
 class GPSSerializer(serializers.Serializer):
+    """GPS data inside a telemetry frame."""
```

Comment:
I guess you describe what a telemetry frame is in DeviceTelemetrySerializer, but i think it is still a bit confusing if you read this in first